src/main/java/cn/nukkit/network/protocol/ProtocolInfo.java

```diff
@@ -17,12 +17,12 @@
     /**
      * Actual Minecraft: PE protocol version
      */
-    int CURRENT_PROTOCOL = dynamic(448);
+    int CURRENT_PROTOCOL = Integer.valueOf("465");
 
     List<Integer> SUPPORTED_PROTOCOLS = Ints.asList(CURRENT_PROTOCOL);
 
-    String MINECRAFT_VERSION = dynamic("v1.17.10");
-    String MINECRAFT_VERSION_NETWORK = dynamic("1.17.10");
+    String MINECRAFT_VERSION = "v1.17.30";
+    String MINECRAFT_VERSION_NETWORK = "1.17.30";
```

Comment:
The protocol version need to be dynamic since otherwise plugins would need to be recompiled against the newer version.